app/controllers/application_controller.rb

```diff
@@ -1,3 +1,4 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  include SessionsHelper
```

Comment:
I believe by default all helpers are included in all controller/view contexts.... I could be wrong and it could be that they are only included in view contexts